robot.py

```diff
@@ -103,32 +102,32 @@ def teleopPeriodic(self):
             self.intake.push(not self.intake.push_on)
```

Comment:
Note: I wasn't able to come up with a good name for the concept of toggling the kicker (toggle_kicker seemed weird to me). I'm not too sure whether this is actually useful either.